test/factory/mix_values_test.dart

```diff
@@ -28,10 +28,13 @@ void main() {
       RandomGenerator.textAttributes(),
     ];
     final variantList = <VariantAttribute>[
-      VariantAttribute(const Variant('testVariant'), RandomGenerator.mix()),
+      VariantAttribute(
+        const Variant('testVariant'),
+        style: RandomGenerator.mix(),
+      ),
       VariantAttribute(
         const Variant('anotherTestVariant'),
-        RandomGenerator.mix(),
+        style: RandomGenerator.mix(),
```

Comment:
Let's change the signature of `VariantAttribute` to a positional parameter, and since its an internal class, we should be fine.